src/main/java/com/softserve/paymentservice/model/Invoice.java

```diff
@@ -0,0 +1,27 @@
+package com.softserve.paymentservice.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Date;
+import java.util.UUID;
+
+@Entity
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Table(name = "invoices")
+public class Invoice {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    private int amount;
```

Comment:
we've discussed it and i remember your point, but i think you have to use BigDecimal here)